src/ontology/edits/oeo-model.omn

```diff
@@ -549,6 +561,75 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1944",
         rdfs:label "has time stamp alignment"@en
     
     
+ObjectProperty: OEO_00390071
+
+    Annotations: 
+        OEO_00020426 "add
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/2013
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2020",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "has study descriptor tag is a relation between a scenario bundle and a oeo class represented by a study descriptor that is tagging it.
+
+Only classes with the oekg annotation property are eligible.",
+        rdfs:label "has study descriptor tag"@en
+    
+    Domain: 
+        OEO_00020227
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000001>
+    
+    
+ObjectProperty: OEO_00390073
+
+    Annotations: 
+        OEO_00020426 "add
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/2013
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2020",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "has scenario type is the relation between a scenario factsheet and the scenario-subclass it is about.",
+        rdfs:label "has scenario type"@en
+    
+    Domain: 
+        OEO_00000365
+    
+    Range: 
+        OEO_00000364
+    
+    
+ObjectProperty: OEO_00390078
+
+    Annotations: 
+        OEO_00020426 "add
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/2013
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2020",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "has reference is the relation between a continuant and a reference that points to a relevant resource to the continuant.",
+        rdfs:label "has reference"@en
+    
```

Comment:
I suggest to add an example of usage annotation with two or three examples (spatial region/country code, document/link to document, ...)